src/Dumplie/Domain/Customer/Product.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Domain\Customer;
+
+use Dumplie\Domain\Customer\Exception\InvalidArgumentException;
+use Dumplie\Domain\SharedKernel\Money\Price;
+
+final class Product
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var Price
+     */
+    private $price;
+
+    /**
+     * @var bool
+     */
+    private $available;
```

Comment:
It depends on what you treat as purchase process. Having such VO as mentioned `Availability` could help with sending supply requests to warehouses in some more complex implementations.